lib/paypal_nvp.rb

```diff
@@ -58,6 +69,9 @@ def call_paypal(data)
       http.verify_mode = OpenSSL::SSL::VERIFY_NONE
     end
 
+    http.open_timeout = @open_timeout
+    http.read_timeout = @read_timeout
```

Comment:
Can we only set them if they have a value, then net/http defaults will apply regardless of what they are? (Future ruby versions may change defaults, etc)